### ISSUE TITLE
[FEAT] unbreak 802.11X for wired connection

### ISSUE THREAD

**Issue by ghost** [secureblue/secureblue#267]
**Describe the benefit this feature would provide to the project**
802.11X is a username password authentication mechanism commonly used on WPA entreprise BUT many forget that wired connection can have this settings enabled in too.

It's my case (my switch require this log to allow wired device to connect (and prevent someone to simply plug his computer to my network)).

The problem is that since it's the same mechanism as WPA it use wpa_supplicant package (package miss from universal blue too) but since this edition is aimed at hardening it I thing unbreack 802.11X for wired connection is in the scope)

**Describe the solution you'd like**
Add wpa_supplicant in image who don't have them (server).

**what this feat is NOT**
This not a call to unblock wifi and will never be as wifi must remain disabled by default.

**Declaration**
- [x] I declare that I have read the README and my feature request is in-scope.

**Comment by ghost** [comment]
I have a settings of Network manager that worked in the base Fedora Silverblue, rebase to universal blue break it, rebase here break it, layer wpa_supplicant have unbreak it (and the search told me that wpa_supplicant is also used for 802.11X)

**Comment by RoyalOughtness** [comment]
Thanks for pointing this out.

When you say universal blue do you mean ucore?

If so, can you open this issue against ucore since this affects all ucore/coreos images and not just secureblue?

https://github.com/ublue-os/ucore

**Comment by ghost** [comment]
> Thanks for pointing this out.
> 
> When you say universal blue do you mean ucore?
> 
> If so, can you open this issue against ucore since this affects all ucore/coreos images and not just secureblue?
> 
> https://github.com/ublue-os/ucore

I try not sure how they will answer trough :)

**Comment by ghost** [comment]
it seem they only have integrated in the ucore and not the ucore minimal in this file : https://github.com/ublue-os/ucore/blob/main/ucore/packages.json with : NetworkManager-wifi

Edit I wait to see if they fix, or not, if they fix by adding NetworkManager-Wifi I warn you so you can edit the file to override remove NetworkManager-Wifi and add simply wpa_supplicant (as wifi is not really in the scope of my issue and I guess not even wanted in securablue server)

**Comment by ghost** [comment]
answer was quite rude, I guess I will continue layering in the meantime.
Thanks for you time tough

**Comment by ghost** [comment]
he stated as won't fix now I guess.
I let you decide if you want to fix sit yourself here or not.

**Comment by RoyalOughtness** [comment]
@BirdInFire the answer they gave was not rude.

The solution here could be to include both the minimal and non minimal ucore images in secureblue.

**Comment by ghost** [comment]
> @BirdInFire the answer they gave was not rude.
> 
> The solution here could be to include both the minimal and non minimal ucore images in secureblue.

the first one was passive agressive, but modified after (seem a mistake) but for answering the solution it's up to you since you are the owner of this project :)

**Comment by RoyalOughtness** [comment]
@BirdInFire He suggested opening a package request. In the meantime I think layering is an appropriate solution here as it would be overkill to add support for `ucore` variants just for a single package.
